llama-index-integrations/embeddings/llama-index-embeddings-ipex-llm/pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.embeddings.ipex_llm"
+
+[tool.llamahub.class_authors]
+IpexLLMEmbedding = "llama-index"
+
+[tool.mypy]
+disallow_untyped_defs = true
+# Remove venv skip when integrated with pre-commit
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.11"
```

Comment:
is 3.11 required? Do i need to upgrade llm integration to 3.11 as well? maybe next pr we need to make an example to use llm and embedding together